packages/docker-node/src/types/index.ts

```diff
@@ -1,9 +1,30 @@
-import type { CellDep } from '@ckb-lumos/lumos'
+import type { CellDep, Indexer, RPC } from '@ckb-lumos/lumos'
 
-export interface DockerNodeStartOptions {
+export interface StartOptions {
+  genesisAccountArgs?: string[]
   port: string
   detached?: boolean
-  genesisAccountArgs?: string[]
+}
+
+export interface DeployOptions {
+  builtInScriptName: string[]
+  configFilePath: string
+  builtInDirPath: string
+  indexer: Indexer
+  rpc: RPC
```

Comment:
The ckb node should know where the rpc endpoint is when deploying the contract, and adding the `lumos` dependency at the interface level feels a bit strange, and how the rpc is called should be put into the concrete implementation. 
Assuming we have a virtual node like evm, we may not use such an rpc to interact

Reply:
These property is not the dependencies for the `Node`, they provider the Lumos config which is from the `Node`.